### PR TITLE
Replay fixes

### PR DESCRIPTION
**Pull request by prsauer**
Screenshot 2023-01-22 at 4 19 02 PM

superset of bigdebuffs pr

### CODE REVIEW

File: packages/shared/src/components/CombatReport/CombatReplay/index.tsx [hunk 1]
```diff
@@ -222,18 +240,25 @@ export function CombatReplay() {
...
-              options={{
-                antialias: true,
-                autoDensity: true,
-                backgroundAlpha: 0,
-              }}
-              onMount={setPixiApp}
-              onUnmount={() => {
-                setPixiApp(null);
-              }}
+            <ContextBridge
```
Comment (kunchenguid) [inline packages/shared/src/components/CombatReport/CombatReplay/index.tsx:243]: interesting... what's special about CombatReplay that makes this necessary?

File: packages/shared/src/components/CombatReport/CombatReplay/ReplayCharacter.tsx [hunk 2]
```diff
@@ -184,30 +184,30 @@ export function ReplayCharacter(props: IProps) {
...
           anchor={{ x: 0.5, y: 0.5 }}
+          x={-3.4}
+          y={-3.0}
         />
       ) : null}
       <ReplayHealthBar current={hp.current} max={hp.max} reaction={props.unit.reaction} />
       <ReplayCastBar unit={props.unit} currentTimeOffset={props.currentTimeOffset} />
-      <ReplaySpellCasts unit={props.unit} currentTimeOffset={props.currentTimeOffset} />
-      <ReplayHpNumbers unit={props.unit} currentTimeOffset={props.currentTimeOffset} />
+      {/* <ReplaySpellCasts unit={props.unit} currentTimeOffset={props.currentTimeOffset} /> */}
```
Comment (kunchenguid) [inline packages/shared/src/components/CombatReport/CombatReplay/ReplayCharacter.tsx:209]: worth keeping this?

Comment (prsauer) [reply packages/shared/src/components/CombatReport/CombatReplay/ReplayCharacter.tsx:209]: I'm going to address these in a follow-up with some kind of GCD display. The Hp numbers might be out though

File: packages/shared/src/components/CombatReport/CombatReplay/index.tsx (hunk 1 above)
Comment (prsauer) [reply packages/shared/src/components/CombatReport/CombatReplay/index.tsx:243]: Contexts aren't passed into custom renderers implicitly I guess -- more at https://reactpixi.org/context-bridge

**Review by kunchenguid — APPROVED** [review]

**Pull request merged**